content/posts/hello-world-dbanalytics.md

```diff
@@ -1,13 +1,45 @@
+
+That has completely changed. The advent of tools like Stitch, Fivetran, dbt, and Snowflake means you can now get up and running very quickly and incredibly cost-effectively. Moreover, you'll have built something that is equipped to scale as your business does. I've spent the last few years [setting up these data stacks in-house for a handful of VC-backed businesses](https://www.growthstreet.co.uk/blog/product/growth-street-bi-stack).
+
+My plan is to work with high-growth businesses, helping them start or scale their use of analytics, putting the right building blocks in place and helping them get value from all the data that is at their disposal. 
+
+I'm really excited about the first few businesses that I'll be working with. If you're interested in having a chat about how your business might be able to make better data-driven decisions, [get in touch!](/contact)
+
+## Moving to Canada 🇨🇦 
+
+A huge factor in my decision is also the fact that I'll be moving to Canada this summer. I moved to the UK from Montreal when I was 13 and am excited to be moving back for a few years. I'll be based in Toronto from later this year.
+
+With the move came a desire for increased flexibility. Freelancing ticked a lot of those boxes. It will hopefully allow me to take a lot of my remote work with me and giving me the time to do a bit of travelling around North and South America from our base in Toronto.
+
+## Writing more
+
+I hope that this post will be the first of many this year. I haven't written much during my time at Growth Street and I've really missed it.
+
+The plan is to write about both the technical and strategic aspects of analytics. My first post will cover the trade-offs between Looker's PDTs and the open-source tool dbt; discussing the reasons why you may want to consider the latter over the former. If that's of interest to you, keep an eye out for it. You can sign up to receive posts via email below.
+
+I also plan to try to do a bit more public speaking this year. I'll be kicking that off at the [Looker Data and Analytics meetup on February 7th](https://www.meetup.com/Looker-London/events/257933863/?_xtd=gqFyqTI1MzYxNTc2M6FwpmlwaG9uZQ&from=ref). I'll be talking a bit about a tool I've been building at Growth Street and in my free time that helps you manage data security in Looker. If you run a data event and are looking for speakers, I'd love to speak to you.
+
+## Thank you
+
+I'm really excited about this next step. 
+
+I want to thank everyone at Growth Street who I have worked with over the past two years. It has been an incredibly fun job and I'm incredibly thankful for the space and support I was given to develop. We are still looking [to hire my replacement](https://growth-street.workable.com/j/7DA67FEE4A), so get in touch if you want to hear a bit more about it.
```

Comment:
Two uses of `incredibly` close to each other – sometimes this works, but here I think you should maybe go with a different word